assets/src/block-validation/amp-validation-status/status-notification.js

```diff
@@ -0,0 +1,107 @@
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import AMPValidationErrorsKeptIcon from '../../../images/amp-validation-errors-kept.svg';
+import { BLOCK_VALIDATION_STORE_KEY } from '../store';
+import { StatusIcon } from '../icon';
+import { SidebarNotification } from '../../block-editor/components/sidebar-notification';
+
+/**
+ * AMP validation status notification component.
+ */
+export default function AMPValidationStatusNotification() {
+	const { autosave, savePost } = useDispatch( 'core/editor' );
+
+	const {
+		ampCompatibilityBroken,
+		fetchingErrorsRequestErrorMessage,
+		hasValidationErrors,
+		isDraft,
+		isEditedPostNew,
+		isFetchingErrors,
+		reviewLink,
+	} = useSelect( ( select ) => ( {
+		ampCompatibilityBroken: select( BLOCK_VALIDATION_STORE_KEY ).getAMPCompatibilityBroken(),
+		fetchingErrorsRequestErrorMessage: select( BLOCK_VALIDATION_STORE_KEY ).getFetchingErrorsRequestErrorMessage(),
+		hasValidationErrors: select( BLOCK_VALIDATION_STORE_KEY ).getValidationErrors()?.length > 0,
+		isDraft: [ 'draft', 'auto-draft' ].indexOf( select( 'core/editor' )?.getEditedPostAttribute( 'status' ) ) !== -1,
+		isEditedPostNew: select( 'core/editor' ).isEditedPostNew(),
+		isFetchingErrors: select( BLOCK_VALIDATION_STORE_KEY ).getIsFetchingErrors(),
+		reviewLink: select( BLOCK_VALIDATION_STORE_KEY ).getReviewLink(),
+	} ), [] );
+
+	if ( isFetchingErrors ) {
+		return null;
+	}
+
+	if ( fetchingErrorsRequestErrorMessage ) {
+		return (
+			<SidebarNotification
+				icon={ <AMPValidationErrorsKeptIcon /> }
+				message={ fetchingErrorsRequestErrorMessage }
+				action={
+					<Button
+						isLink
+						onClick={ isDraft
+							? () => savePost( { isPreview: true } )
+							: () => autosave( { isPreview: true } )
+						}
+					>
+						{ __( 'Try again', 'amp' ) }
+					</Button>
+				}
+			/>
+		);
+	}
+
+	if ( ampCompatibilityBroken ) {
+		return (
+			<SidebarNotification
+				icon={ <AMPValidationErrorsKeptIcon /> }
+				message={ __( 'AMP blocked from validation issues marked kept.', 'amp' ) }
+				action={ reviewLink && (
+					<ExternalLink href={ reviewLink }>
+						{ __( 'View technical details', 'amp' ) }
+					</ExternalLink>
+				) }
+			/>
+		);
+	}
+
+	if ( hasValidationErrors ) {
+		return (
+			<SidebarNotification
+				icon={ <StatusIcon broken={ true } /> }
+				message={ __( 'AMP is working, but issues needs review.', 'amp' ) }
+				action={ reviewLink && (
+					<ExternalLink href={ reviewLink }>
+						{ __( 'View technical details', 'amp' ) }
+					</ExternalLink>
+				) }
+			/>
+		);
+	}
+
+	if ( isEditedPostNew ) {
+		return (
+			<SidebarNotification
+				icon={ <StatusIcon /> }
+				message={ __( 'Validation issues will be checked for when the post is saved.', 'amp' ) }
+			/>
+		);
+	}
+
+	return (
+		<SidebarNotification
+			icon={ <StatusIcon /> }
+			message={ __( 'AMP is working. All issues reviewed or removed.', 'amp' ) }
```

Comment:
Similar to above:

```suggestion
			message={ __( 'AMP is enabled. All issues have been reviewed.', 'amp' ) }
```

cc @westonruter

Reply:
Actually, I think this should be split into two. If there are no validation errors at all, then the message should be: “AMP is enabled. There are no validation issues.”

Otherwise, if there are validation issues but they have all been reviewed, it should be: “AMP is enabled. All validation issues have been reviewed.”

Reply:
Done in 8b42ad96e385431f71f561818126ef8d0fcd8cd9